src/deprecated/deprecated.js

```diff
@@ -115,6 +115,8 @@ import { basisInitialize } from '../resources/basis.js';
 import { EventHandler } from '../core/event-handler.js';
 import { Asset } from '../asset/asset.js';
 
+import { VrManager } from '../vr/vr-manager.js';
```

Comment:
I actually think we should just strip out WebVR support completely. https://caniuse.com/webvr

Reply:
ah that bad? Ok, maybe I can do that in a separate PR? Are there any old projects that depend on it? I assume not, considering it would not work in a browser at all.

Reply:
https://github.com/playcanvas/engine/issues/4176